backend/x86_binary_emitter.ml

```diff
@@ -1151,6 +1151,40 @@ let emit_movmskpd b dst src =
       emit_mod_rm_reg b 0 [ 0x0f; 0x50 ] rm (rd_of_reg64 reg)
   | _ -> assert false
 
+let emit_vex3 buf ~rexr ~rexx ~rexb ~vexm ~vexw ~vexv ~vexl ~vexp =
```

Comment:
(Same remark as in the target branch:
if we have no tests for this code, it seems
very easily for both the developer and
the reviewer to let a typo slip.)

Reply:
This is tested by compiling scalar_ops.ml with the binary emitter